app/plugins/oauth2-authentication.md

```diff
@@ -79,6 +79,7 @@ form parameter                                    | default | description
 `config.accept_http_if_already_terminated`<br>*optional* | `false` | Accepts HTTPs requests that have already been terminated by a proxy or load balancer and the `x-forwarded-proto: https` header has been added to the request. Only enable this option if the Kong server cannot be publicly accessed and the only entry-point is such proxy or load balancer.
 `config.anonymous`<br>*optional*           | `` | An optional string (consumer uuid) value to use as an "anonymous" consumer if authentication fails. If empty (default), the request will fail with an authentication failure `4xx`
 `config.global_credentials`<br>*optional*   | `false` | An optional boolean value that allows to use the same OAuth credentials generated by the plugin with any other API whose OAuth 2.0 plugin configuration also has `config.global_credentials=true`.
+`config.refresh_token_ttl`<br>*optional*   | `1209600` | An optional integer value telling the plugin how many seconds a token/refresh token pair is stored before it's removed. Default value is 2 weeks. Set to `0` to keep the token/refresh token pair forever.
```

Comment:
I think the terminology "before it's being removed" refers to the technical implementation, not the functional effect. So something like "expiration" or "validity" might be a better terminology here.

Reply:
Gotcha, let me reword it.